CoEpi/repo/CoepiRepo.swift

```diff
@@ -70,11 +71,26 @@ class CoEpiRepoImpl: CoEpiRepo {
                 }
                 if !matchedKeys.isEmpty {
                     os_log("Matches found for keys: %@", log: servicesLog, type: .debug, "\(matchedKeys)")
+
+                    //Show notification that matches were found (background)
+                        //Needs to open alert view
+                    let content = UNMutableNotificationContent()
+                    content.title = "New Contact Alerts"
+                    content.body = "New contact alerts have been detected. Tap for details."
+                    content.sound = UNNotificationSound.default
+                    let trigger = UNTimeIntervalNotificationTrigger(timeInterval: 1, repeats: false)
+                    let request = UNNotificationRequest(identifier: UUID().uuidString, content: content, trigger: trigger)
+                    UNUserNotificationCenter.current().add(request)
```

Comment:
Put this code in a component (e.g. `ReportsNotificationShower`), have it observe the reports (`CoEpiRepo.reports`) and trigger the notification from there. Remember also to initialize the component with .eagerSingleton type, since if it's not referenced from other places, it will not become active otherwise.